internal/pkg/describe/service.go

```diff
@@ -296,6 +300,32 @@ func (d *ecsServiceDescriber) ServiceConnectDNSNames() ([]string, error) {
 	return service.ServiceConnectAliases(), nil
 }
 
+// DeploymentConfigAlarmNames returns the rollback alarm names of a service.
+func (d *ecsServiceDescriber) DeploymentConfigAlarmNames() ([]string, error) {
+	service, err := d.ecsClient.Service(d.app, d.env, d.service)
+	if err != nil {
+		return nil, fmt.Errorf("get service %s: %w", d.service, err)
+	}
+	if service.DeploymentConfiguration.Alarms == nil {
+		return nil, nil
+	}
+	return aws.StringValueSlice(service.DeploymentConfiguration.Alarms.AlarmNames), nil
+}
+
+// DeploymentType returns 'Alarm-Based Rollback' or 'Default'.
+func (d *ecsServiceDescriber) DeploymentType() (string, error) {
+	deploymentType := "Default"
+	var err error
+	alarmNames, err := d.DeploymentConfigAlarmNames()
+	if err != nil {
+		return "", fmt.Errorf("retrieve alarm names: %w", err)
+	}
+	if alarmNames != nil {
```

Comment:
It doesn't make a lot of sense to me that if the deployment alarm name is specified and the deployment type becomes "Alarm-Based Rollback" 🤔  "Default" is also ambiguous term if it's not defined by ECS. Maybe we don't need to make changes to this section?

Reply:
Is alarm based rollback exclusive with "Rolling"? 

From my understanding, alarm-based rollback is an addition on top of "Rolling":
1. All services in Copilot is [rolling](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/deployment-type-ecs.html)
2. We have "default" rolling and "recreate" rolling
3. Alarm-based rollback is not exclusive with any of the above. That is, we can have "default rolling + alarm-based rollback", "default rolling w/o alarm-based rollback", "recreate rolling + abr", "recreate rolling w/o abr"

If this is the case, I agree that returning either "Rolling" or "Alarm-Based Rollback" might be a little counterintuitive here.

Reply:
In the long run I think we need a better visual representation for all these settings:
|Environment | test | prod|
|:-----------|:----|:-----|
| Count | 1 | Autoscaling 1-10|
| vCPU | 0.25 | 0.25 |
| Mem | 512 | 512 |
| Deployment Strategy | recreate | rolling with min 100% and max 200% |
| Rollback on alarm | ❌ | ✔️ |

Reply:
Okay, for now, we'll let the listed Rollback Alarms indicate that that's the deployment type. I'll remove the column.

Reply:
<img width="951" alt="Screen Shot 2023-02-15 at 4 33 19 PM" src="https://user-images.githubusercontent.com/60631893/219227569-ffb3e6a1-cbe3-4f39-b929-24c970fbff30.png">